lib/WorseReflection/Core/Inference/Resolver/StringLiteralResolver.php

```diff
@@ -36,17 +36,20 @@ public function resolve(NodeContextResolver $resolver, Frame $frame, Node $node)
     private function getStringContentsText(StringLiteral $node): string
     {
         $children = $node->children;
+        if (is_array($children)) {
+            $children = $children[0];
+        }
+
```

Comment:
For some reason the node's children is an array for when it's a here doc comment. I don't know why though.

Reply:
yes it's a bit broken at the parser level. can we also check that `[0]` exists?